### PR TITLE
Convert maps to arrays

### PR DESCRIPTION
**Pull request by fabianTMC**

### CODE REVIEW

File: test/transform.coffee [hunk 1]
```diff
@@ -58,4 +58,26 @@ describe 'ObjectTemplate', ->
...
+          what = new json2json.ObjectTemplate { path: 'sportsTeams', key: false, mapToArray: true, as: false}
+            .transform json
+            .should.deep.equal [['yankees', 'New York Yankees', [ 'Alex', 'Starlin' ]], ['cubs', 'Chicago Cubs', 'Jason']]
+
+        it 'should extract only the specified properties of each map into a nested array', ->
+          what = new json2json.ObjectTemplate { path: 'sportsTeams', key: false, mapToArray: true, as: false, choose: ['id']}
+            .transform json
+            .should.deep.equal [['yankees'], ['cubs']]
+
+        it 'should extract only the specified properties of each map into a nested array', ->
```
Comment (joelvh) [inline test/transform.coffee]: @fabianTMC the description looks like a copy/paste - sould be "flattened array"

**Comment by joelvh** [comment]
@fabianTMC thanks for the contribution! Just saw a minor thing, but looks good to me.

**Comment by fabianTMC** [comment]
Yes, that was a copy-paste issue. Just fixed it.

**Comment by joelvh** [comment]
@fabianTMC thanks for the updates. Released v0.2.7 to NPM.

**Pull request merged**